packages/neon_framework/lib/src/widgets/account_tile.dart

```diff
@@ -29,27 +28,24 @@ class NeonAccountTile extends StatelessWidget {
   /// {@endtemplate}
   final Account account;
 
+  final UserDetailsBloc userDetailsBloc;
+
+  /// {@macro neon_framework.UserStatus.userStatusBloc}
+  final UserStatusBloc? userStatusBloc;
```

Comment:
No, because `NeonProvider.of<UserStatusBloc>(context)` will always return the bloc of the active account and not necessarily of the account that is displayed here.